### PR TITLE
Do not ignore imported assemblies

### PR DESCRIPTION
**Pull request by theoxuan**

### CODE REVIEW

**Comment by shiftkey** [comment]
Not being familiar with Unity, is there a compelling reason to track binary assemblies by default? Any supporting links or discussions would be greatly appreciated...

**Comment by theoxuan** [comment]
It's just a matter of directories. Binary assemblies under the directory of **Assets/** should not be ignored by default.

**Comment by shiftkey** [comment]
@theoxuan any other links discussing this, which I can point other curious to?

File: Unity.gitignore [hunk 1]
```diff
@@ -14,6 +14,9 @@
 *.pidb
 *.booproj
 
+# Do not ignore imported assemblies
+!*.dll
```
Comment (shiftkey) [inline Unity.gitignore]: > Binary assemblies under the directory of **Assets/** should not be ignored by default.

So this rule could be made more specific?

**Comment by theoxuan** [comment]
@shiftkey Hello here is the official documentation about Unity3D folder structures. 
[https://unity3d.com/learn/tutorials/modules/beginner/architecture/folders-in-version-control](url)

**Comment by shiftkey** [comment]
@theoxuan that's great, thanks for sharing!

I'm still not quite clear on why we're interested in tracking these `*.dll` assemblies by default though...

**Comment by theoxuan** [comment]
@shiftkey A huge bunch of third-party plugins are used in Unity projects. Some of them are source codes while others are assemblies. By default, all assemblies in Assets/ folder should not be ignored, but those in Libray/ and Temp/ folders.

**Comment by JaySoyer** [comment]
Just wanna chime in and mention that @theoxuan is correct.  We need to track dll files under the Assets/ folder only.  However, in regards to the PR...I'm not sure we need to explicitly mention to not ignore dll files since it's already providing the correct behavior as written.

**Comment by shiftkey** [comment]
@JaySoyer thanks for following up. 

@theoxuan as mentioned above, there doesn't seem to a rule here which is preventing assemblies from being tracked in a Git repository - do we need to do this?

**Pull request closed without merging**